src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,24 @@
-export const PostInfo = () => <>Put the post here</>;
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <>
```

Comment:
looks like a redundant fragment 

https://react.dev/reference/react/Fragment#usage

Reply:
Still not fixed